packages/node_modules/@webex/internal-plugin-encryption/src/kms.js

```diff
@@ -662,6 +662,7 @@ const KMS = WebexPlugin.extend({
       clientId: originalContext.clientInfo.clientId,
       credential: {
         userId: onBehalfOf,
+        onBehalfOf, // Supports running onBehalfOf self. i.e. A CO which calls onBehalfOf with CO.id.
```

Comment:
Previously this field was depreciated. To resolve this bug KMS are re-using it. There is a bit of redundancy here and the API is less user friendly. But KMS report that the alternative is a more complex item of work, with a higher risk.